nodes/cma/nodes-cma-autoscaling-custom.adoc

```diff
@@ -49,3 +49,21 @@ Successfully updated ScaleTarget
 ----
 
 You can temporarily pause the autoscaling of a workload object, if needed. For example, you could pause autoscaling before performing cluster maintenance.
+
+.Custom metrics autoscaler workflow
+image::564_OpenShift_Custom_Metrics_Autoscaler_0224.png[Custom metrics autoscaler workflow]
+
+1. A user creates or modifies a scaled object custom resource for a workload on the cluster. The object contains the scaling configuration for that workload. Prior to accepting the new object, the OpenShift API server sends it to the custom metrics autoscaler admission webhooks process to ensure that the object is valid. If validation succeeds, the API server persists the object.
+
```

Comment:
Line 58: (or, data sources)

IBM [style guide](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses) discourages using parenthetical comments in general text.

Reply:
I agree that the parentheses should be omitted.

Reply:
@rh-tokeefe @bburt-rh I know. I just couldn't think of a better way to do it.  How about:

> the controller monitors any external trigger sources, also known as data sources, that are specified in the object for changes to the metrics data